src/main/resources/init.sql

```diff
@@ -0,0 +1,15 @@
+DROP SCHEMA IF EXISTS CAESAR;
+CREATE SCHEMA CAESAR CHARACTER SET UTF8;
+USE CAESAR;
+
+
+CREATE USER IF NOT EXISTS 'admin'@'localhost' IDENTIFIED BY '1234%';
+GRANT ALL PRIVILEGES ON *.* TO 'admin'@'localhost';
```

Comment:
Since we have actual ```root``` user on db host, there is no need to grant full access to separate user. Other two accounts should be enough.